app.base.json

```diff
@@ -32,6 +32,8 @@
     "footer1": "Bel ons dan",
     "footer2": "op werkdagen van 9.00 tot 17.00 uur.",
     "headerTitle": "",
+    "helpText": "Bereikbaar van maandag tot en met vrijdag van 08:00 tot 18:00 uur.",
```

Comment:
Availability is already in `footer2` prop of the `language` settings

Reply:
configuration.language.footer2 is not used anywhere. Perhaps better to remove footer2, since the text is no longer in the footer anyway?

Reply:
The same goes for `footer1`.

Reply:
Markdown can now be used in `helpText`, right?

Reply:
yes, markdown is used